docs/features/map-tiles-custom-areas/high-level-design.md

```diff
@@ -1,43 +1,76 @@
+access to the `planning_areas` table in the marxan geoprocessing database. For
+visualization purposes, we only need to access to the `the_geom` column.
 
-The tiler system will follow a similar implementation as the other tiler systems within this project. In the geoprocessing service a custom query will be used to access the `the_geom` column of the `planning_areas` table and to generate the tiles.
-The data upload jobs should provide back the id of the planning area id. this information should be use to query the `the_geom` column of the `planning_areas` table. This will then be served to the user as a proxy of the mvt tile in the api service.
+The tiler system will follow a similar implementation as the other tiler systems
+within this project. In the geoprocessing service a custom query will be used to
+access the `the_geom` column of the `planning_areas` table and to generate the
+tiles. The data upload jobs should provide back the id of the planning area id.
+this information should be use to query the `the_geom` column of the
+`planning_areas` table. This will then be served to the user as a proxy of the
+mvt tile in the api service.
 
 ## Preview custom planning grid tiler
 
-In order to generate the vector tiles for the custom planning grid, we need to access to the `planning_units_geom` table in the marxan geoprocessing database. For visualization purposes, we only need to access to the `the_geom` column.
+In order to generate the vector tiles for the custom planning grid, we need to
+access to the `planning_units_geom` table in the marxan geoprocessing database.
+For visualization purposes, we only need to access to the `the_geom` column.
 
-The tiler system will follow a similar implementation as the other tiler systems within this project. In the geoprocessing service a custom query will be used to access the `the_geom` column of the `planning_units_geom` table and to generate the tiles.
-The data upload jobs should provide back the id of the planning area id, by default `project_id` column in the `planning_units_geom` will be populated with the id of the **custom uploaded planning area**. this information should be use to query the `the_geom` column of the `planning_units_geom` table. This will then be served to the user as a proxy of the mvt tile in the api service.
+The tiler system will follow a similar implementation as the other tiler systems
+within this project. In the geoprocessing service a custom query will be used to
+access the `the_geom` column of the `planning_units_geom` table and to generate
+the tiles. The data upload jobs should provide back the id of the planning area
+id, by default `project_id` column in the `planning_units_geom` will be
+populated with the id of the **custom uploaded planning area**. this information
+should be use to query the `the_geom` column of the `planning_units_geom` table.
+This will then be served to the user as a proxy of the mvt tile in the api
+service.
```

Comment:
```suggestion
populated with the id of the **custom uploaded planning area**. This information
should be use to query the `the_geom` column of the `planning_units_geom` table.
This will then be served to the user via a proxy to the mvt tiler in the api
service.
```